tcping.go

```diff
@@ -227,7 +227,7 @@ func usage() {
 }
 
 // setPrinter selects the printer
-func setPrinter(tcping *tcping, outputJSON, prettyJSON *bool, outputDb *string, args []string) {
+func setPrinter(tcping *tcping, outputJSON, prettyJSON *bool, timeStamp *bool, outputDb *string, args []string) {
```

Comment:
by the way, this is why CI is complaining about complex code. Although, I do not expect you to solve this.

Reply:
oh okk thanks for the info